be/src/env/env_posix.cpp

```diff
@@ -590,9 +591,26 @@ class PosixEnv : public Env {
     }
 
     // Create the specified directory. Returns error if directory exists.
-    Status create_dir(const std::string& name) override {
-        if (mkdir(name.c_str(), 0755) != 0) {
-            return io_error(name, errno);
+    Status create_dir(const std::string& dir_path) override {
```

Comment:
This interface is to create a directory, not create a directory recursively.
And create directory recursively should be implemented in file_utils, because it's the same logic for different types of Env.

Reply:
Thanks, will modify it.